### ISSUE TITLE
Cargos para trust

### ISSUE THREAD

**Issue by ItsGabii** [PerfectDreams/StickyNotes#32]
## 🤔 Antes de enviar...
Para manter tudo organizado, siga estes passos antes de começar:
* Eu já confirmei que esta sugestão será útil para outros usuários.
* Eu já procurei por sugestões parecidas no repositório.
* Eu já verifiquei que não existe nenhuma outra branch ou pull request aberto com a minha sugestão ou com uma sugestão parecida.
* Eu tentei procurar por [outros repositórios do PerfectDreams](https://github.com/PerfectDreams) um repositório relacionado com a minha sugestão, mas não encontrei/é uma sugestão sobre algo que não tem nada relacionado no servidor.

### 🎮 Seu nome no PerfectDreams
*Its_Gabi*

### ✨ Descrição da sugestão
Pelo que eu me lembre o plugin de proteção usado (GriefPrevention) usa uma espécie de cargos para limitar o player de fazer coisas erradas no terreno do outro. Mas praticamente ninguém usa esse sistema, e gostaria de melhorar ele.

 trust1: mexer em alavancas e botões
 trust2: alterar blocos
 trust3: mexer em baús, craftingtables, fornalhas e suportes de poções
trust 4:geral

### 🌟 Como irá funcionar?
Dependendo de cada situação o player escolhe o trust que ele vai dar a tal player

/trust alavancas (pessoa) para trust1 ou /ta (pessoa)
/trust bloco (pessoa) para trust2 ou /tb (pessoa)
/trust container (pessoa) para trust3 ou /tc (pessoa)
/trust geral (pessoa) para trust4 ou simplesmente /trust (pessoa)

### 😎 Porque ela seria legal no PerfectDreams?
Muitos players são roubados por dar trust para alguém que vai fazer algum serviço como destruir montanhas, portanto isso seria proteção contra roubo.

**Comment by FlammDumbFox** [comment]
Não sei se é necessário, o GriefPrevention já tem trusts o suficiente, IMO.

Se não me engano, em ordem de nível de acesso, trust de acesso, trust de containers (botões, chests, animais, farms, ...), trust de blocos (modificar o claim) e trust de permissão (possibilitando o usuário a dar /trust para outros jogadores sem que o mesmo seja o dono do terreno.)

Eu pessoalmente acho que isso tudo já é suficiente.

**Comment by MrPowerGamerBR** [comment]
Sim, o GriefPrevention já tem essa funcionalidade, basta explicar para os jogadores como elas funcionam.
